components/single_image_view/single_image_view.jsx

```diff
@@ -23,6 +23,7 @@ const PREVIEW_IMAGE_MIN_DIMENSION = 50;
 
 export default class SingleImageView extends React.PureComponent {
     static propTypes = {
+        post: PropTypes.object.isRequired,
```

Comment:
May pass `postId` only if no other use for `post` object.

Reply:
I thought about it but was concerned about changing as guess the effects of this. I thought it  was added intentionally on the parent for rendering with changes to post object. 
I changed it to have just postId hopefully it is just an error previously.